### PR TITLE
Add split locked

### LINKED ISSUE #3671: Expenditures: As a user I can display a locked state
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to display state where split funds are locked.

## Acceptance Criteria

- an option to display and click an edit icon
- an option to display total amount of funds and currency
- an option to display recipient and his share with currency
- an option to display updated log event
- an option to display updated stage status
- equal/unequal toggle should not be visible

## Design

- [Figma Link - Split locked state](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3746%3A315142)

download (61)

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds locked state of split payment.

Screenshot 2022-08-24 at 12 06 10

[Figma Link - Split locked state](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=3746%3A315142)

## TODO

- add edition in locked state after adding this [PR](https://github.com/JoinColony/colonyDapp/pull/3609)

Resolves #3671

### CODE REVIEW

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Solid work! I just noticed some small things:
FireShot Capture 699 - Create Expenditure - Colony - wonker - localhost
FireShot Capture 700 - Create Expenditure - Colony - wonker - localhost

1. The recipient's containers look like they have different padding for the top and bottom, so it doesn't look centered.
2. We could use more space between the token icon and the amount, currently it's `4px`, if we use `8px` it could look closer to the specs.
3. The amount label doesn't look aligned to me, but only by like `1px` - `2px`. Might be screen/OS but it's noticeable
4. With really long usernames it goes out of the container. We should establish a width in which we should cut off the name with an ellipsis, like how we do it in other places in the dapp.

**Review by arrenv — APPROVED** [review]
Looks good to me @agnieszkajarosikloj

**Review by chinins — APPROVED** [review]
Well done!

File: src/modules/dashboard/components/ExpenditurePage/Split/LockedSplit.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                        token={token}
+                        name={token.name || token.address}
+                      />
+                      <Numeral
+                        unit={getTokenDecimalsWithFallback(0)}
+                        value={recipient.amount || 0}
+                      />
+                      {token.symbol}
+                    </div>
+                  </>
```
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ExpenditurePage/Split/LockedSplit.tsx:110]: The unnecessary empty bracket are still here

**Review by ArmandoGraterol — APPROVED** [review]
Excellent work! Just noticed that the empty brackets are still there, but other than that detail, this is good to go

**Pull request merged**